### ISSUE TITLE
Storage and gas usage calculators

### ISSUE THREAD

**Issue by ilblackdragon** [near/near-explorer#555]
People don't really understand the storage costs and gas calculations.

If would be good to have an embedded caculator, where based on:
 - bytes -> shows how much $N it will use. + show standard size for a single account (100b) / access key (82b) etc.
 - Tgas -> how much $N it will use. It's not changing right now, but people are used to estimating.

**Comment by bowenwang1996** [comment]
> It's not changing right now

I don't think that is completely accurate. When bridge catches up for example, it often resulted in nearly full blocks.
